examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -74,18 +74,18 @@
      "text": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.                      pre=mm.SequentialBlock([mm.SequenceMaskLastInference(), mm.ReplaceMaskedEmbeddings()]),
</pre><p>Radek do mind to add one liner explanation about SequenceMaskLastInference.  why we added it and what it does.. you can say simply write something like " SequenceMaskLastInference() -  is set to support inference. The input sequence and its corresponding mask should be extended by one position at the end to account for the next-item (target) position. To do this, we  set SequenceMaskLastInference as a pre-layer of&nbsp;<code>ReplaceMaskedEmbeddings()</code>&nbsp;using the sequential-block." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/943/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
hi @rnyak! Thank you for the comment! 🙂 I integrated this piece of information with what we already had there, please see below

Could you please verify if this is looking okay? Thank you!

![image](https://user-images.githubusercontent.com/2444926/211412804-52d3a9e8-5bee-4e68-9148-4fd2b93531c8.png)

Reply:
@radekosmulski it is not really selecting last item in the sequence. it is extending the original input sequence  and its corresponding mask  by one position at the end to account for the next-item (`target`) position at the `inference step` so not at the model.fit().  I recommend you to remove that arg since it is not required for model training and evaluation. Better to remove it :)

Reply:
that is a great point! apologies, didn't realize that this is how it works, remove it now

Reply:
<p>thanks! I am really sorry for back and forth. I think since you removed the&nbsp;<code>SequenceMaskLastInference</code>&nbsp;you can set the XLNETBlock like that, similar to what you were originally doing.</p><pre class="ql-syntax" spellcheck="false">mm.XLNetBlock(d_model=dmodel, n_head=4, n_layer=2, pre=mm.ReplaceMaskedEmbeddings())
</pre><p> </p>